js_modules/dagit/packages/core/src/graph/PipelineGraph.stories.tsx

```diff
@@ -104,7 +104,6 @@ export const Basic = () => {
 
   return (
     <PipelineGraph
-      backgroundColor={ColorsWIP.White}
```

Comment:
Guessing these were already being drawn over a bg white so they dont need their own style explicitly?

Reply:
Yep this hasn't been used since the big redesign a while back I think 👍